src/redux/categories/categories.js

```diff
@@ -0,0 +1,16 @@
+const STATUS = 'src/categories/STATUS';
```

Comment:
- [ ] (OPTIONAL) Kindly correct this as it should be `const STATUS = 'bookstore/categories/STATUS';` :+1: